tools/windows/DatadogAgentInstaller/WixSetup/localization-en-us.wxl

```diff
@@ -47,7 +47,7 @@
 	<String Id ="DDAgentUserDialogDescription" Overridable="yes" Localizable="yes">{\WixUI_Font_Normal_White}Enter the agent user account</String>
 	<String Id ="DDAgentUserDialogTitle"  Overridable="yes" Localizable="yes">{\WixUI_Font_Title_White}Datadog Agent User Account</String>
 	<String Id ="DDAgentUserDialogUserNameLabel" Overridable="yes" Localizable="yes">Please enter the username (leave blank to use ddagentuser):</String>
-	<String Id ="DDAgentUserDialogPasswordLabel" Overridable="yes" Localizable="yes">Please enter the password (leave blank to use random password):</String>
+	<String Id ="DDAgentUserDialogPasswordLabel" Overridable="yes" Localizable="yes">Please enter the password (leave blank to re-use existing password if user exists or use random password if not):</String>
```

Comment:
hmm this is tough. for domain accounts the password will be re-used, but local accounts will have their password reset. I don't want to explain every case in the installer UI though...

Reply:
line is too long
![image](https://user-images.githubusercontent.com/2266830/225792106-ee3e0414-d94c-4bd1-b53b-9643d6989274.png)

Reply:
Any way you can link to documentation from this box? It does seem like a difficult amount of information to capture here.

Reply:
Perhaps "Existing users: Enter your password. New users: Enter a new password"

Reply:
Good point. It should be fine - we usually add redirects if we move something